cluster/examples/kubernetes/ceph/csi/rbd/snapshotclass.yaml

```diff
@@ -3,12 +3,12 @@ apiVersion: snapshot.storage.k8s.io/v1beta1
 kind: VolumeSnapshotClass
 metadata:
   name: csi-rbdplugin-snapclass
-driver: rook-ceph.rbd.csi.ceph.com
+driver: rook-ceph.rbd.csi.ceph.com # provisioner:namespace:system
 parameters:
   # Specify a string that identifies your cluster. Ceph CSI supports any
   # unique string. When Ceph CSI is deployed by Rook use the Rook namespace,
   # for example "rook-ceph".
   clusterID: rook-ceph
   csi.storage.k8s.io/snapshotter-secret-name: rook-csi-rbd-provisioner
-  csi.storage.k8s.io/snapshotter-secret-namespace: rook-ceph
+  csi.storage.k8s.io/snapshotter-secret-namespace: rook-ceph # namespace:cluster
```

Comment:
```suggestion
  csi.storage.k8s.io/snapshotter-secret-namespace: rook-ceph # namespace:system
```

Reply:
I'd like to have @Madhu-1 weigh in here. I was pretty sure that the driver is in the system/cluster namespace but the parameters are unique for each ceph cluster.

Reply:
This comment is in `storageclass.yaml`
```yaml

  # The secrets contain Ceph admin credentials. These are generated automatically by the operator
  # in the same namespace as the cluster.
  csi.storage.k8s.io/provisioner-secret-name: rook-csi-cephfs-provisioner
  csi.storage.k8s.io/provisioner-secret-namespace: rook-ceph # namespace:cluster
  csi.storage.k8s.io/controller-expand-secret-name: rook-csi-cephfs-provisioner
  csi.storage.k8s.io/controller-expand-secret-namespace: rook-ceph # namespace:cluster
  csi.storage.k8s.io/node-stage-secret-name: rook-csi-cephfs-node
  csi.storage.k8s.io/node-stage-secret-namespace: rook-ceph # namespace:cluster
```

Reply:
Yes, the drivers will be per operator and the secrets are per ceph clusters.

Reply:
thanks for the clarification